test/cppsuite/test_harness/debug_utils.h

```diff
@@ -0,0 +1,27 @@
+#ifndef DEBUG_UTILS_H
+#define DEBUG_UTILS_H
+
+#include <cstdint>
+#include <iostream>
+#include <string>
+
+/* Define helpful functions. */
+namespace test_harness {
+
+#define DEBUG_ABORT -1
+#define DEBUG_ERROR 0
+#define DEBUG_INFO 1
+
+static int64_t _trace_level = 0;
+
+/* Used to print out traces for debugging purpose. */
+static void
+debug_info(const std::string &str, int64_t trace_threshold, int64_t trace_level)
+{
+    if (trace_threshold >= trace_level)
```

Comment:
Could we use `_trace_level` here instead of passing it from everywhere?

Reply:
It is usually not good practice to rely on global vars, especially when it comes to unit testing. I suggest we could keep it as it is.